doc/contributor/how-to/contribute-layers.md

```diff
@@ -86,3 +86,38 @@ A correct QGreenland data pipeline will output data that:
   * In GeoPackage (`.gpkg`) format.
   * Uses the `label` attribute name for pre-calculated labels when using
     generic styles with labels, for example `labeled_point.qml`
+
+
+## Layer quality checklist
+
+
+### All layers
+
+- Abstract populated
+- Citation populated
+- Layer descriptions populated
+- Layer description excludes data provider
+- Layers'/groups' title follows date standard
+- Layers'/groups' title follows capitalization standard
+- Layers'/groups' title excludes data provider (see notes for exceptions)
```

Comment:
Can we link to the mentioned notes?

Reply:
I'm not sure. I'm going to see if I can find them in Jira? Maybe they're in the repo someplace, too.

Reply:
FOUND IT! https://docs.google.com/spreadsheets/d/13qzH-HGaWz0wqKbpG_9yoNpWy7K8UagSUbdQ-XS12mU/edit#gid=0

Reply:
`8a2831b7` populates info from the original spreadsheet notes